imap_processing/swe/l1a_swe.py

```diff
@@ -0,0 +1,183 @@
+import bitstring
+import numpy as np
+import xarray as xr
+
+from imap_processing import packet_definition_directory
+from imap_processing.swe import decom_swe
+
+
+def uncompress(cem_count):
+    """This function uncompress a count.
+
+    Parameters
+    ----------
+    cem_count : int
+        CEM counts. Eg. 243
+
+    Returns
+    -------
+    int
+        uncompressed count. Eg. 40959
+    """
+    # index is the first four bits of input data
+    # multi is the last four bits of input data
+    index = cem_count // 16
+    multi = cem_count % 16
+
+    # uncompression formula
```

Comment:
I'd suggest linking to explicit pages/locations in the algorithm documents in these formulas too. I think that will help us later going back to debug things and cross-reference check where values came from.